Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Filters.kt

```diff
@@ -79,31 +85,49 @@ fun FilterChip(
 ) {
     val (selected, setSelected) = filter.enabled
     val backgroundColor by animateColorAsState(
-        if (selected) JetsnackTheme.colors.brand else JetsnackTheme.colors.uiBackground
+        if (selected) JetsnackTheme.colors.brandSecondary else JetsnackTheme.colors.uiBackground
     )
     val border = Modifier.fadeInDiagonalGradientBorder(
         showBorder = !selected,
         colors = JetsnackTheme.colors.interactiveSecondary,
         shape = shape
     )
     val textColor by animateColorAsState(
-        if (selected) JetsnackTheme.colors.textInteractive else JetsnackTheme.colors.textSecondary
+        if (selected) Color.Black else JetsnackTheme.colors.textSecondary
     )
+
     JetsnackSurface(
-        modifier = modifier
-            .height(28.dp)
-            .then(border),
+        modifier = modifier.height(28.dp),
         color = backgroundColor,
         contentColor = textColor,
         shape = shape,
         elevation = 2.dp
     ) {
+        val interactionSource = remember { MutableInteractionSource() }
+
+        val pressed by interactionSource.collectIsPressedAsState()
+        val backgroundPressed =
+            if (pressed)
+                Modifier.offsetGradientBackground(
+                    JetsnackTheme.colors.interactiveSecondary,
+                    200f,
+                    0f
+                )
+            else
+                Modifier.background(Color.Transparent)
```

Comment:
Sorry, the comment about braces was requesting *curly* braces around the clauses e.g.

```suggestion
        val backgroundPressed =
            if (pressed) {
                Modifier.offsetGradientBackground(
                    JetsnackTheme.colors.interactiveSecondary,
                    200f,
                    0f
                )
            }  else {
                Modifier.background(Color.Transparent)
            }
```

Same for other changes.